sources/Inputs/src/NetworkManager.cpp

```diff
@@ -102,7 +102,7 @@ NetworkManager::buildTree() {
       auto targetP = (*it_g)->getTargetP();
       auto pmin = (*it_g)->getPMin();
       auto pmax = (*it_g)->getPMax();
-      if ((*it_g)->isVoltageRegulationOn() && (DYN::doubleEquals(targetP, pmin) || targetP > pmin) && (DYN::doubleEquals(targetP, pmax) || targetP < pmax)) {
+      if ((*it_g)->isVoltageRegulationOn() && (DYN::doubleEquals(-targetP, pmin) || -targetP > pmin) && (DYN::doubleEquals(-targetP, pmax) || -targetP < pmax)) {
```

Comment:
@marcochiaramello is that what you expected here ? Thks!

Reply:
yes!